Product/Production/Gateway/DocumentQuery_20/src/main/java/gov/hhs/fha/nhinc/docquery/_20/nhin/DocQueryImpl.java

```diff
@@ -49,6 +50,6 @@ public AdhocQueryResponse respondingGatewayCrossGatewayQuery(AdhocQueryRequest b
             assertion.setImplementsSpecVersion(UDDI_SPEC_VERSION.SPEC_2_0.toString());
         }
 
-        return inboundDocQuery.respondingGatewayCrossGatewayQuery(body, assertion);
+        return inboundDocQuery.respondingGatewayCrossGatewayQuery(body, assertion, this.getWebContextProperties(context));
```

Comment:
Right here (and in the equivalent 3.0 class) is the only place you're using "this" where a field isn't shadowed, is this intentional?